app/components/link_text_component.rb

```diff
@@ -10,6 +10,7 @@ def initialize(text:, icon: nil, target: nil)
 
   def call
     svg_icon = !@icon&.empty? ? inline_svg(@icon) : ''
-    "#{@text}<span class='mx-1'>#{svg_icon}<span>".html_safe
```

Comment:
This a generic component, and can't contain specific details about only mappings.
bellow a more genir solution, that may work:

```ruby 
    svg_icon = !@icon&.empty? ? inline_svg(@icon) :  nil
    extra_span = svg_icon ? nil : "<span class='mx-1'>#{svg_icon}</span>"
    "#{@text}#{extra_span}".html_safe
```

Reply:
@syphax-bouazzouni 
I tried using a similar solution, but it removes the icon in 'See all metadata'
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/31127782/65b1618a-fa4d-4b0c-829e-23f1ca59d483)
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/31127782/5ac7f127-10e0-4702-9ee2-b7853a1b1c60)